english/Enter-a-dim-reality/Dim_reality.md

```diff
@@ -0,0 +1,8 @@
+Something happened after you called the magic spell "Alohomora".  You keep your eyes closed, afraid to open them - anticipating a shift to an even stranger place.  Slowly you open them.  Wait..  Are they open?  You blink several times, but there is only darkness.  You move your hands across your body making sure that you still exits, physically.  You muster the courage to take a step into the blackness, it seems that you are on solid ground - for the moment.  After taking several steps you realize that the ground you are walking on has some give, and there is a brushing sound as you shuffle your feet.  You slowly reach beneath you and are suprised to find grass and earth.  It is cool to the touch, slightly damp.  You dig your fingers deep into the soft soil, and are suddenly overwhelmed with a sense of comfort and belonging.  Something echoes in the distance, straining your ears you frantically try to find the source of the sound on the dark horizon.  A dim light comes into focus, its hard to tell how far off.  Get to the light. Your timid steps quickly turn into a jog, the cool moist air is invigorating.  The light doesn't seem to be getting any closer, you begin to run faster.  Eventually you are out of breath, and you stop to rest.  The light is still on the horizon, perhaps a bit closer, perhaps not.  As you gear up for continued pursuit, you notice another light, in the opposite direction, the direction you came from.  
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.